CHANGELOG_PENDING.md

```diff
@@ -19,7 +19,9 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
 - [types] \#4582 Block: `Round` is now int32
 - [consensus] \#4582 RoundState: `Round`, `LockedRound` & `CommitRound` are now int32
 - [consensus] \#4582 HeightVoteSet: `round` is now int32
+- [evidence] \#4617 Remove `Pubkey` from DuplicateVoteEvidence
```

Comment:
why is it breaking again?

Reply:
amino encoding would change. some nodes in a network would have the full type and some wouldn't I believe this would cause issues. If not then I can make the changes to merge it to master

Reply:
That may be true. I guess the only way is to check 😛

Reply:
it is breaking. Removing a field in amino would change the field number. If we try to decode on a 0.33 deployed net it will fail.